.travis.yml

```diff
@@ -1,8 +1,13 @@
 language: c
 
-os:
-  - linux
-  - osx
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+    - os: linux
+      compiler: clang
+    - os: osx
```

Comment:
In the past, travis didn't allowed a project to test on mac and on linux. Can you confirm that it worked somewhere?